lib/types/string/ip.js

```diff
@@ -29,12 +41,14 @@ internals.Ip.createIpRegex = function (versions, cidr) {
     for (let i = 0; i < versions.length; ++i) {
         const version = versions[i];
         if (!regex) {
-            regex = '^(?:' + internals.Ip.versions[version];
+            regex = '^(?:' + internals.Ip.versions[version] + internals.Ip.cidrs[version][cidr];
```

Comment:
Caught a bug here where it was creating an incorrectly large RegExp by appending to the string twice for each version supported.  It's not really much of a performance gain for the validation phase, but does provide us with some gains in the compilation phase of the RegExp.